jax/_src/numpy/ufuncs.py

```diff
@@ -2555,10 +2555,15 @@ def _float_divmod(x1: ArrayLike, x2: ArrayLike) -> tuple[Array, Array]:
   mod = lax.rem(x1, x2)
   div = lax.div(lax.sub(x1, mod), x2)
 
-  ind = lax.bitwise_and(mod != 0, lax.sign(x2) != lax.sign(mod))
+  ind = lax.bitwise_and(mod != 0, signbit(x2) != signbit(mod))
   mod = lax.select(ind, mod + x2, mod)
+  mod = _where(signbit(x2) != signbit(mod), negative(mod), mod)
```

Comment:
Can we use `mod = copysign(mod, x2)` here?

Also, here and below, it's slightly more efficient to use `lax.neg` rather than `negative`; the latter does a bunch of validation that has already been done in this function.

Reply:
Thanks! `copysign` works here. replaced `negative` with `lax.neg` in the other place.